src/gridstack.js

```diff
@@ -379,7 +383,8 @@
         this.opts = _.defaults(opts || {}, {
             width: parseInt(this.container.attr('data-gs-width')) || 12,
             height: parseInt(this.container.attr('data-gs-height')) || 0,
-            item_class: 'grid-stack-item',
+            item_class: opts.item_class,
```

Comment:
There should be a default class, here what you're doing is that you're enforcing to pass item_class, and if it's not passed it will throw an error. Same for line below \/